processor.go

```diff
@@ -140,24 +137,39 @@ func (p *Processor) Run() {
 				return
 			}
 
+			if buildJob == nil {
```

Comment:
Just out of curiosity, when would this happen (since we're actually crafting the buildJob object before it ends up on the buildJobsChan)?

Reply:
You're right that this shouldn't happen.  I think it's a leftover from an earlier iteration where I was trying to do a ready channel at the processor level.